### DISCUSSION TITLE
Wireguard help

### DISCUSSION THREAD

**Discussion by supsm** [celzero/rethink-app#1054] [Q&A] [answered]
Hi all, I've been trying to get wireguard set up for rethinkdns. I generated a keypair from the app, added the server as a peer and added the client ip to the server... etc. It just doesn't work for some reason. I also installed the wireguard android app, repeated the steps with a new keypair and ip address, and it works (I can connect to internet and my ip displays the server's ip). So I don't think my setup has any issue. I added all apps to the rethinkdns wireguard profile, and requests do show up as "proxied" within the logs. When I run `ifconfig` in termux, I see that `tun4` has inet 10.111.222.1. On the wireguard app, `tun4` has inet 10.8.0.3, which corresponds to my configured address. Not sure what is going wrong, but thanks nonetheless for wireguard integration.

**Comment by supsm** [comment, marked as answer]
Nevermind, I figured it out. I had to set `allowed ips` to `0.0.0.0/0` instead of `10.8.0.0/24`. Silly me.

**Reply by ignoramous** [reply]
Also, if you're using IPv6 WireGuard tunnles, set *Configure* -> *Network* -> *Choose IP version* to *Auto* AND make sure the underlying network supports IPv6.
